src/core/stream.c

```diff
@@ -713,7 +731,86 @@ QuicStreamParamGet(
 
         Status = QUIC_STATUS_SUCCESS;
         break;
+    case QUIC_PARAM_STREAM_BLOCKED_TIMINGS: {
+
+        if (*BufferLength < sizeof(QUIC_FLOW_BLOCKED_TIMING) * QUIC_FLOW_BLOCK_REASON_COUNT) {
+            *BufferLength = sizeof(QUIC_FLOW_BLOCKED_TIMING) * QUIC_FLOW_BLOCK_REASON_COUNT;
+            Status = QUIC_STATUS_BUFFER_TOO_SMALL;
+            break;
+        }
+
+        if (Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
 
+        uint64_t Now = CxPlatTimeUs64();
```

Comment:
We should probably return `QUIC_STATUS_INVALID_STATE` for this if not `Started`.